WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -433,22 +435,28 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String permissi
                 if (permissions == null || permissions.length == 0) {
```

Comment:
Hah, true! Missed that `@NonNull` type decoration already in the method signature :). Will remove the null-check happily!